operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md

```diff
@@ -1,24 +1,98 @@
+* Minimize content so it can be found and remembered. Keep pages short, modular, and focused on a single topic.
+
+## Writing Style
+
+**Numbers**
+
+Spell out numbers when the number is the first word in a sentence or is less than or equal to ten, otherwise use numeric digits. Use commas to make long numbers easier to read.
+
+**Capitalization**
+
+Use title case for page titles and section titles.
+
+Title case: Grammar, Spelling, and Mechanics
+Sentence case: Language and spelling
+
+## Avoid Fluff
+
+We avoid using the following words and phrases as they're overused and vague:
+
+* Work better
+* Do better work, faster
+* Get more done in less time
+* Chat (as a stand alone feature reference. However it is ok to use chat when describing a benefit "Create a Jira ticket without leaving your chat window" )
+* DevOps teams 
+* Utilize (instead, use “use”)
+* High performance teams
+* Phrases that directly praise ourselves: “We’ve built an intuitive workplace messaging solution”, “It’s a joy to use”
+
+## Writing an MVP
+
+When you submit a PR for your contribution, please include documentation with it. The documentation can be submitted as a new file using the following format:
+
+\## Introduction and Context\
+*Details of the current feature/function and what the contribution fixes or adds. This can be presented as bullet points.*
+
+\## Configuration (if applicable)\
+*Provide configuration and set up steps in the context of the change/feature.*
+
+\## Code Samples (if applicable)\
+*Include any code samples that should be included in the user-facing documentation.*
+
+\## Installation Process\
+*Include any processes or procedures that users have to perform to enable/use the change/addition.*
+
+\## Troubleshooting/Known Issues (if applicable)\
+*Include any known issues or limitations that may affect how the feature/process works.*
+
+We don't expect a huge body of documentation or that it's perfectly-written - but rather a clear, concise outline of the change which can be added to our documentation. 
+
+During the editorial review, editors may make punctuation and/or terminology changes and commit them to save time on the review process. This only applies to punctuation/terminology - any content suggestions and questions will follow the usual review and discussion process.
+
+Once the editorial review is complete, we'll move your documentation contribution to the appropriate part of the guide (if it's not there already). We'll share the URL and you can edit it at any time if you need to.
+
+## More Information
 
 These guidelines were inspired by [Stripe's knowledge base content](https://document360.io/blog/tear-down-of-stripe-knowledge-base/), which is widely perceived as excellent by the developer community, and [by other sites shared in this Document360 article](https://document360.io/blog/10-knowledge-base-software-best-practice-examples/).
```

Comment:
This is related to my changes, but re-reading it, wondering if we need to say `widely perceived as excellent` and just keep it simple:

```
These guidelines were inspired by [Stripe's knowledge base content](https://document360.io/blog/tear-down-of-stripe-knowledge-base/) and [by other sites shared in this Document360 article](https://document360.io/blog/10-knowledge-base-software-best-practice-examples/).
```

Reply:
Done.